kubernetes-examples/README.md

```diff
@@ -44,7 +44,7 @@ where `${kubernetes_example_directory}` is replaced by a path to an example dire
 To use an alternative image for Kroxylicious, set the `KROXYLICIOUS_IMAGE` environment variable.
 
 ```shell
-KROXYLICIOUS_IMAGE=quay.io/kroxylicious/kroxylicious-developer:0.4.0 ./scripts/run-example.sh ${kubernetes_example_directory}
+./scripts/run-example.sh ${kubernetes_example_directory}
```

Comment:
It looks like this snippet is supposed to demonstrate customizing the `KROXYLICIOUS_IMAGE`

Reply:
yeah, I've modified the wrong file!